corehq/apps/app_manager/suite_xml.py

```diff
@@ -2399,9 +2401,7 @@ def get_commands():
                                     case = None
                             else:
                                 case = SESSION_CASE_ID.case()
-
-                            if case:
-                                command.relevant = interpolate_xpath(form.form_filter, case)
+                            command.relevant = interpolate_xpath(form.form_filter, case)
```

Comment:
I still don't think this logic is quite right. If the user creates a basic form and puts `#case` in the filter expression but they  don't load a case then the app will crash because the filter expressions will be looking for a `case_id` session variable that isn't in the session.